<resources>
    <!--全局信息-->
    <string name="error">ERROR</string>
    <string name="yes">Yes</string>
    <string name="no">No</string>

    <!--Crash崩溃捕捉活动-->
    <string name="crash_title_text">Crash Log</string>
    <string name="crash_content_text">Ah! An uncontrollable exception occurred in the app. The detailed log is shown below, \nwhich can be copied and fed back to the developer. \nWe will try our best to repair the exception. Thank you for your cooperation. \nEvery feedback will make the app better!</string>
    <string name="crash_error">An exception from an unknown source has been detected and the log cannot be captured normally. \n you can dictate the crash replication process to the developer \ n</string>

</resources>